README.md

```diff
@@ -7,31 +7,35 @@
 [![PyPI version](https://badge.fury.io/py/sdrf-pipelines.svg)](https://badge.fury.io/py/sdrf-pipelines)
 ![PyPI - Downloads](https://img.shields.io/pypi/dm/sdrf-pipelines)
 
-The SDRF pipelines provide a set of tools to validate and convert SDRF files to different workflow configuration files such as MSstats,OpenMS and MaxQuant.
+**Validate and convert SDRF files with sdrf-pipelines and its `parse_sdrf` CLI.** 
 
-### Installation
+This is the official SDRF file validation tool and the and convert SDRF files to different workflow configuration files such as MSstats, OpenMS and MaxQuant.
```

Comment:
```suggestion
This is the official SDRF file validation tool and it can convert SDRF files to different workflow configuration files such as MSstats, OpenMS and MaxQuant.
```